charts/tekton-pipelines/values.yaml

```diff
@@ -108,7 +108,43 @@ buildpacks:
             fi
           securityContext:
             privileged: true
+      postDeployTaskSteps:
+        - name: argo-events
+          image: badouralix/curl-jq
+          imagePullPolicy: IfNotPresent
+          resources: {}
+          script: |
+            #!/usr/bin/env sh
+
+            # add extra params from env and convert them to lowercase to work with this
+            # data later in argo workflow
+            EXTRA=$(jq -n env | jq 'walk(if type=="object" then with_entries(.key|=ascii_downcase) else . end)')
+            JSON_TEMPLATE='{
+              "project": "$(params.project)",
+              "environment": "$(params.environment)",
+              "application": "$(params.application)",
+              "sha": "$(params.sha)",
+              "extra": %s
+            }'
+            JSON_PAYLOAD=$(printf "$JSON_TEMPLATE" "$EXTRA")
+
+            echo "Payload: $JSON_PAYLOAD"
+
+            OUTPUT=$(curl -s -o /dev/null -w httpcode=%{http_code} --location \
+              --request POST "build-succeed-eventsource-svc.argo-events.svc.cluster.local:12000/build-succeed" \
+              --header 'Content-Type: application/json' \
+              --data-raw "$JSON_PAYLOAD")
+
+            STATUS_CODE=$(echo "${OUTPUT}" | sed -e 's/.*\httpcode=//')
+            if [ ${STATUS_CODE} -ne 204 ] && [ ${STATUS_CODE} -ne 200 ]; then
+                echo "Curl operation/command failed due to server return code - ${STATUS_CODE}"
+                exit 1
+            fi
+
+            echo "Sent 'build-succeed' webhook"
 
+          securityContext:
+            privileged: true
```

Comment:
Why do we need it privileged? (same question to other pipelines)

Reply:
done